fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -1815,7 +1817,34 @@ private Partition createPartitionWithIndices(String clusterName, long dbId, long
             }
 
             if (!ok || !countDownLatch.getStatus().ok()) {
-                errMsg = "Failed to create partition[" + partitionName + "]. Timeout:" + (timeout / 1000) + " seconds.";
```

Comment:
Moreover, I think we could treat `!ok` as timeout, and `!countDownLatch.getStatus().ok()` as BE down.